README.md

```diff
@@ -43,6 +43,11 @@
   <td>2018-04-18</td>
   <td>2019-04-18</td>
 </tr>
+<tr>
+  <td>N/A</td>
```

Comment:
I know this is a LTS-specific page, but I would really be in favour of writing `Current` here. It makes distinguishing v5’s status from v7’s easier + it’s what the schedule diagram below says.